earthpy/plot.py

```diff
@@ -0,0 +1,471 @@
+"""
+The ``earthpy`` spatial module provides functions that wrap around ``rasterio``
+and ``geopandas`` to work with raster and vector data in Python.
+"""
+
+import numpy as np
+import numpy.ma as ma
+import matplotlib.pyplot as plt
+from matplotlib import patches as mpatches
+from matplotlib.colors import ListedColormap
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import rasterio as rio
+from skimage import exposure
+import earthpy.spatial as es
+
+
+def colorbar(mapobj, size="3%", pad=0.09, aspect=20):
+    """Adjusts the height of a colorbar to match the axis height. Note that
+    this function will not work properly using matplotlib v 3.0.0 in Jupyter
+    or when exporting an image. Be sure to update to 3.0.1.
+
+    Parameters
+    ----------
+    mapobj : the matplotlib axes element.
+    size : char
+        The percent width of the colorbar relative to the plot. default = 3%
+    pad : int
+        The space between the plot and the color bar. Default = .09
+
+    Returns
+    -------
+    matplotlib.pyplot.colorbar
+
+        Matplotlib color bar object with the correct width that matches the
+        y-axis height.
+
+    Examples
+    --------
+
+    .. plot::
```

Comment:
sphinx -- tell it that the following code chunk produces a plot. And it should be plotting it.... without it the figure won't render.